rasa/validator.py

```diff
@@ -244,9 +267,28 @@ def verify_nlu(self, ignore_warnings: bool = True) -> bool:
         stories_are_valid = self.verify_utterances_in_stories(ignore_warnings)
         return intents_are_valid and stories_are_valid and there_is_no_duplication
 
+    def verify_form_slots(self) -> bool:
+        """Verifies that form slots match the slot mappings in domain."""
+        domain_slot_names = [slot.name for slot in self.domain.slots]
+        everything_is_alright = True
+
+        for f in self.domain.form_names:
```

Comment:
optional: I'm in favor of spelling it out to make the code more readable
```suggestion
        for forms in self.domain.form_names:
```